WebDriverAgentLib/Utilities/FBXPath.m

```diff
@@ -530,6 +535,20 @@ + (NSString *)valueForElement:(id<FBElement>)element
 
 @end
 
+@implementation FBAccessibleAttribute
```

Comment:
should we mix this together in a single PR? I'd rather add the attribute in the other PR (and updated integration tests for it)

Reply:
Yeah, this was just for debugging for now, but this part should be separated when we add _accessible_ attribute.